### PR TITLE
O biekt2

### PR DESCRIPTION
**Pull request by Ryniek88**
teraz powinno byc dobrze

### CODE REVIEW

File: src/Garnek.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+public class Garnek {
```
Comment (ghost) [inline src/Garnek.java]: please use English nomenclature

File: src/Garnek.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+        int high;
+        int width;
+        String color;
+
+       public Garnek(){
+       }
```
Comment (ghost) [inline src/Garnek.java]: The number of empty lines between elements should be constant so please add an empty line between these two constructors and remove reduntant empty lines below second constructor

File: src/OOP.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+public class OOP {
+    public static void main(String[] args) {
+     //   Garnek garnek = new Garnek();
+     //   garnek.high = 3;
+     //   garnek.width = 5;
+     //   garnek.color = "black";
```
Comment (ghost) [inline src/OOP.java]: Please remove the commented code or uncomment it if necessary - we do not leave comments in the code

File: src/OOP.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
```
Comment (ghost) [inline src/OOP.java]: Same -> reduntant empty lines. Only one is enough.

File: src/Garnek.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
+public class Garnek {
+
+        int high;
```
Comment (ghost) [inline src/Garnek.java]: These fields should be private but you are probably not familiar with access modifiers yet, that's okay. If you're interested in learning more about it, here's a link -> https://javastart.pl/baza-wiedzy/java-podstawy-jezyka/modyfikatory-dostepu

**Review by Ryniek — CHANGES_REQUESTED** [review]

**Review by Ryniek — APPROVED** [review]
please work on the tabs in the code next time because they are not consistent

**Pull request merged**